packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/payment-methods-list.jsx

```diff
@@ -43,42 +43,39 @@ const PaymentMethodsList = () => {
                     >
                         <Localize i18n_default_text='Add new' />
                     </Button>
-                    <ThemedScrollbars className='payment-methods-list__list' height={'60vh'}>
```

Comment:
can we remove `payment-methods-list__list` if it's not needed elsewhere?

Reply:
@farrah-deriv I checked this class in styles files and there are no styles defined for `payment-methods-list__list` itself.